### PR TITLE
chore: package pipeline spec into a standalone Python package

### PR DESCRIPTION
**Pull request by chensun**
**Description of your changes:**
Package pipeline_spec into a standalone Python package.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: api/v2alpha1/python/setup.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import subprocess
+import sys
+import setuptools
+try:
+  from distutils.spawn import find_executable
+except ImportError:
+  from shutil import which as find_executable
+
+NAME = "kfp-pipeline-spec"
+VERSION = "0.0.1"
```
Comment (chensun) [inline api/v2alpha1/python/setup.py:25]: For a proof-of-concept/demo purpose, I use 0.0.1 for now. Will update once finalized package name, etc.

Comment (chensun) [reply api/v2alpha1/python/setup.py]: changed to 0.0.2 with metadata update (adding license)

File: api/v2alpha1/python/kfp_pipeline_spec/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Google LLC
```
Comment (neuromage) [inline api/v2alpha1/python/kfp_pipeline_spec/__init__.py:1]: Can we ensure the namespace is `kfp.pipeline_spec` instead of `kfp_pipeline_spec` ?

Comment (chensun) [reply api/v2alpha1/python/kfp_pipeline_spec/__init__.py]: Updated.

**Pull request merged**